src/Psr/Tracing/RelayOpenTelemetry.php

```diff
@@ -112,9 +112,13 @@ public function __call(string $name, array $arguments)
             return $this->relay->{$name}(...$arguments);
         }
 
+        $operation = strtoupper($name);
+        $stmt = $this->fmtCommand($operation, $arguments);
+
         $span = $this->tracer->spanBuilder('Relay::' . strtolower($name))
-            ->setAttribute('db.operation', strtoupper($name))
             ->setAttribute('db.system', 'redis')
+            ->setAttribute('db.operation', $operation)
+            ->setAttribute('db.statement', $stmt)
```

Comment:
This won't work with `__construct()` and `connect()` and some other, which aren't Redis commands, but Relay specific methods.

Reply:
"This" is `db.statement`? Why it does work for `db.operation`?

Reply:
That's also sending invalid data currently. We're working on the stubs to make this easier.

Reply:
This will make it easier to identify various types of methods: https://relay.so/api

Reply:
Okay, sounds like I should wait for some input from you.

Reply:
@vmihailenco: We're exploring whether we can use 8.0 attributes in the class stubs. If so we could use reflection to check whether a method should be bypassed (`setOption()`), should be traced (`connect()`) or whether it's a Redis command (`mget()`).

Instead of maintaining arrays of functions we could parse all class methods once, or each method once when it's executed.

Reply:
Thanks for update. Let me know what changes do you expect here.

My 5 cents are that I would not worry about this. I'd say that tracing `connect` is the right thing to do and having `setOption` also would not hurt. Are you worried about having `db.statement` on such spans? Again, I don't see what real issues that causes.,,